### PR TITLE
Housing system

### PR DESCRIPTION
**Pull request by madiyarkoishibekov**
Please be kind to review my changes

Best regards, 
Madiyar Koishibekov

### CODE REVIEW

**Review by barys21 — APPROVED** [review]

File: UtopiaCity/Controllers/HousingSystem/HousingSystemController.cs [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                Residents = residents.Select(x => x.FirstName + " " + x.FamilyName).ToList()
+            };
+
+            var items = new List<SelectListItem>();
+            foreach (string accountNames in viewModel.Residents)
+            {
+                items.Add(new SelectListItem() { Text = $"{accountNames}", Value = $"{accountNames}" });
+            }
+            ViewBag.SelectList = items;
+            //List<SelectListItem> items = new List<SelectListItem>();
```
Comment (crocomoth) [inline UtopiaCity/Controllers/HousingSystem/HousingSystemController.cs]: commented code?

File: UtopiaCityTest/Controllers/HousingSystem/HousingSystemControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
+﻿//using AutoMapper;
+//using Microsoft.AspNetCore.Mvc;
+//using Microsoft.EntityFrameworkCore;
+//using Moq;
+//using System;
+//using System.Collections.Generic;
```
Comment (crocomoth) [inline UtopiaCityTest/Controllers/HousingSystem/HousingSystemControllerTests.cs]: ?

**Review by crocomoth — CHANGES_REQUESTED** [review]

File: UtopiaCity/Controllers/HousingSystem/HousingSystemController.cs (hunk 1 above)
Comment (madiyarkoishibekov) [reply UtopiaCity/Controllers/HousingSystem/HousingSystemController.cs]: Forgot to delete

File: UtopiaCityTest/Controllers/HousingSystem/HousingSystemControllerTests.cs (hunk 2 above)
Comment (madiyarkoishibekov) [reply UtopiaCityTest/Controllers/HousingSystem/HousingSystemControllerTests.cs]: This controller test had some issues which I have resolved just now

**Pull request closed without merging**